pdns/distributor.hh

```diff
@@ -181,7 +183,7 @@ template<class Answer, class Question, class Backend>MultiThreadDistributor<Answ
 // start of a new thread
 template<class Answer, class Question, class Backend>void MultiThreadDistributor<Answer,Question,Backend>::distribute(int ournum)
 {
-  setThreadName("pdns/distributo");
+  setThreadName("pdns/distributor");
```

Comment:
Pieter would know but I'm pretty sure we truncated this because there wasn't any space left.

Reply:
correct!